server/routes/referrals/referralsController.ts

```diff
@@ -43,15 +49,66 @@ export default class ReferralsController {
   }
 
   async createReferral(req: Request, res: Response): Promise<void> {
-    const referral = await this.interventionsService.createDraftReferral(res.locals.user.token)
+    let error: FormValidationError | null = null
```

Comment:
Can we put a comment here explaining why this code is in the controller and not in a form object (presumably, because it's temporary)? I think the inconsistency will be confusing otherwise. I mean, ideally, we'd just put it in a form object and not worry about it further. Because temporary code can often end up not being temporary.

Reply:
Have moved this into a form, now.